ale/base/type_sensor.py

```diff
@@ -115,7 +115,7 @@ def ephemeris_time(self):
           ephemeris times split based on image lines
         """
 
-        return np.arange(self.ephemeris_start_time + (.5 * self.exposure_duration), self.ephemeris_stop_time + self.interframe_delay, self.interframe_delay)
+        return np.arange(self.ephemeris_start_time, self.ephemeris_stop_time, self.interframe_delay)
```

Comment:
I imagine this was just changed to match ISIS?

Reply:
Somewhat, rather than apply the `+ (.5 * self.exposure_duration)` in the pushframe sensor type, the start time should be updated on the driver.